ui/admin/config/features.js

```diff
@@ -28,7 +28,7 @@ const baseEdition = {
   'target-network-address': false,
   'vault-worker-filter': false,
   'ldap-auth-methods': false,
-  'dynamic-credentials-worker-filter': false,
```

Comment:
we have a few worker filter flags, for now I only consolidated the ones that we use for host catalogs, I can add a ticket to look into other worker filters in our backlog

Reply:
https://hashicorp.atlassian.net/browse/ICU-16190 added here